tests/integration.rs

```diff
@@ -5490,6 +5490,45 @@ mod tests {
                     fname
                 ))
             }
+
+            fn wait_for_boot(&self) -> bool {
+                let tmo_max = 180;
+                let tmo = 5;
+                let mut retries = u8::MAX;
+                let mut elapsed = 0;
+                let cmd = "dir /b c:\\ | find \"Windows\"";
+                loop {
+                    if elapsed + tmo >= tmo_max {
+                        // Last iteration.
+                        retries = 0;
+                    }
+
+                    let out = std::panic::catch_unwind(|| {
+                        match ssh_command_ip_with_auth(
+                            cmd,
+                            &self.auth,
+                            &self.guest.network.guest_ip,
+                            retries,
+                            tmo,
+                        ) {
+                            Ok(out) => out,
+                            Err(_e) => "".to_string(),
+                        }
+                    })
+                    .unwrap();
+
+                    if "Windows" == out.trim() {
+                        println!("ELAPSED={}", elapsed);
+                        return true;
+                    }
+
+                    elapsed += tmo;
+                    if elapsed >= tmo_max {
+                        break;
+                    }
+                }
+                false
+            }
```

Comment:
Since `ssh_command_ip_with_auth` already has a retry strategy, could we keep this whole thing simple by relying on what `ssh_command_ip_with_auth` provides?

Reply:
Ok, i reworked this function to keep up with the timeout increase approach in `ssh_command_ip_with_auth` but made sure the timeout wouldn't grow too fast so then boot can be caught quickier.

Thanks